compiler/luci/pass/src/FoldDequantizePass.cpp

```diff
@@ -43,7 +46,10 @@ bool is_foldable_const(luci::CircleConst *node)
 
 luci::CircleConst *dequantized_const_node(luci::CircleConst *const_node)
 {
-  assert(const_node->quantparam() != nullptr);
+  if (const_node->quantparam() == nullptr)
+  {
+    throw std::runtime_error("Given constant node cannot be dequantized");
```

Comment:
How about adding some more information about why this node can't be dequantized,, like `the node doesn't have a quant-related parameter~`.